src/Chapter1.hs

```diff
@@ -520,7 +522,12 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y
+    | x == 0 || y == 0 = 0
+    | x > 0 && y > 0 = min x y
+    | x > 0 && y < 0 = if x < (y * (-1)) then x else y
+    | y > 0 = if (x * (-1)) < y then x else y
+    | otherwise = if x > y then x else y
```

Comment:
This implementation looks almost correct, but it's a bit more complicated than it could be. I recommend using the [abs](http://hackage.haskell.org/package/base-4.14.1.0/docs/GHC-Num.html#v:abs) function to simplify the logic here